### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by saschagrunert**
- github.com/Microsoft/go-winio: fc70bd9a86b5 → 3fe6c5262873
- github.com/containerd/project 7fb81da5e663 → github.com/containerd/release-tool 1e44fc26031c
- github.com/containerd/ttrpc: 0e0f228740de → 4f1b8fe65a5c
- github.com/containernetworking/cni: v0.7.1 → 83439463f784
- github.com/containers/buildah: 20e92ffe0982 → v1.13.1
- github.com/containers/libpod: 04e8bf3dba50 → v1.7.0
- github.com/containers/ocicrypt: b87a4a69c741 → 142388cb70de
- github.com/containers/storage: e18327c57107 → v1.15.7
- github.com/cri-o/ocicni: 7762645d18ca → deac903fd99b
- github.com/docker/docker: ada3c14355ce → f9d136b6c68d
- github.com/godbus/dbus: 8a1682060722 → 4481cbc300e2
- github.com/gogo/protobuf: 65acae22fc9d → v1.3.1
- github.com/kubernetes/kubernetes: v1.18.0-alpha.1 → v1.18.0-alpha.2
- github.com/onsi/ginkgo: v1.10.3 → v1.11.0
- github.com/onsi/gomega: v1.7.1 → v1.8.1
- github.com/opencontainers/image-spec: 775207bd45b6 → 9ea04d1f37d7
- github.com/opencontainers/selinux: 5215b1806f52 → 00892252b298
- github.com/pkg/errors: v0.8.1 → v0.9.1
- github.com/prometheus/client_golang: v1.2.1 → v1.3.0
- github.com/seccomp/containers-golang: v0.3.1 → v0.3.2
- github.com/urfave/cli/v2: v2.0.0 → v2.1.1
- github.com/vishvananda/netlink: v1.0.0 → v1.1.0
- golang.org/x/net: 13f9640d40b9 → 6afb5195e5aa
- golang.org/x/sys: 63cb32ae39b2 → 34d275377bf9
- google.golang.org/grpc: v1.25.1 → v1.26.0
- k8s.io/utils: 6ca3b61696b6 → 5f6fbceb4c31

### CODE REVIEW

File: docs/crio.8.md [hunk 1]
```diff
@@ -243,16 +241,12 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--read-only**: Setup all unprivileged containers to run as read-only. Automatically mounts tmpfs on `/run`, `/tmp` and `/var/tmp`. (default: false)
 
-**--registries-conf**="": path to the registries.conf file
-
```
Comment (saschagrunert) [inline docs/crio.8.md]: Fixed by urfave/cli to not document hidden flags any more :)

**Comment by mrunalp** [comment]
Closing in favor of #3148

**Pull request closed without merging**